api/src/main/java/io/opentelemetry/trace/propagation/B3Propagator.java

```diff
@@ -37,7 +39,7 @@
  * href=https://github.com/openzipkin/b3-propagation>openzipkin/b3-propagation</a>.
  */
 @Immutable
-public class B3Propagator implements HttpTextFormat<SpanContext> {
+public class B3Propagator implements HttpTextFormat {
```

Comment:
Open an issue to move B3 in a contrib package, before we release beta. We keep only w3c in the api.

Reply:
Sounds good to me.

Reply:
Link?

Reply:
#1035